src/components/footer/example-footer-cymraeg.njk

```diff
@@ -1,7 +1,3 @@
----
-'fullWidth': true
----
-
```

Comment:
Was this change inherited from my PR? Just because I thought Id leave this change for now

Reply:
Yeah I think revert it

Reply:
Other than this it looks good